fullCalendar/fullCalendar.d.ts

```diff
@@ -8,27 +8,6 @@
 
 declare namespace FullCalendar {
     export interface Calendar {
-
-        /**
-         * Formats a Date object into a string.
-         */
-        formatDate(date: Date, format: string, options?: Options): string;
-
-        /**
-         * Formats a date range (two Date objects) into a string.
-         */
-        formatDates(date1: Date, date2: Date, format: string, options?: Options): string;
-
-        /**
-         * Parses a string into a Date object.
-         */
-        parseDate(dateString: string, ignoreTimezone?: boolean): Date;
-
-        /**
-         * Parses an ISO8601 string into a Date object.
-         */
-        parseISO8601(dateString: string, ignoreTimezone?: boolean): Date;
```

Comment:
Not in version 2.
On 20 Jun 2016 2:07 pm, "Mohamed Hegazy" <notifications@github.com> wrote:

> In fullCalendar/fullCalendar.d.ts
> <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/9602#discussion_r67766623>
> :
>
> > -        formatDate(date: Date, format: string, options?: Options): string;
> > -
> > -        /**
> > -         * Formats a date range (two Date objects) into a string.
> > -         */
> > -        formatDates(date1: Date, date2: Date, format: string, options?: Options): string;
> > -
> > -        /**
> > -         * Parses a string into a Date object.
> > -         */
> > -        parseDate(dateString: string, ignoreTimezone?: boolean): Date;
> > -
> > -        /**
> > -         * Parses an ISO8601 string into a Date object.
> > -         */
> > -        parseISO8601(dateString: string, ignoreTimezone?: boolean): Date;
>
> i do see these in the public API documentation.
>
> http://fullcalendar.io/docs1/utilities/parseISO8601/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/9602/files/9c49c0460b7a221bbff9dde1edd336f26977d282#r67766623>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFEgkUEI-BUVzMYw-KVezsYgxIip2VK8ks5qNwEWgaJpZM4IxqUp>
> .
>